### PR TITLE
Fix#15027 Made appropriate placement of colons and info link as suggested

### LINKED ISSUE #15027: Colon in index.php file
### Is your feature request related to a problem? Please describe.
I have attached an image below, in which you can see there's no need for a colon.

image

### Describe the solution you'd like
Remove the colon from the UI, because UI doesn't need that.

### Additional context

- phpMyAdmin version: 4.8.5

### PR DESCRIPTION
**Pull request by kartik1000**
Signed-off-by: Kartik Kathuria <kathuriakartik0@gmail.com>

### Description

The placement of colons and info link was suggested to be changed in #15027, Therefore made the appropriate changes.

Fixes #15027

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

### CODE REVIEW

**Comment by williamdes** [comment]
@sudo-bot /deploy

**Comment by AghaSaad04** [comment]
Sir @williamdes, Isn't this somewhat related to #15027 That colon placement thing.

**Comment by kartik1000** [comment]
@sudo-bot /deploy

**Comment by kartik1000** [comment]
@williamdes, Thanks for reviewing, Hey Now I think that all colons are in place but  i also made changes adding colon in front of language and moving that info button after language to after dropdown as @ibennetch said but that can't be seen from this PR because it is not showing Language option at all

**Comment by kartik1000** [comment]
Screenshot 2019-04-12 at 7 42 24 PM
This is the final view.

**Comment by shucon** [comment]
Multiple failed tests, refer: https://travis-ci.org/phpmyadmin/phpmyadmin/builds/519262787?utm_source=github_status&utm_medium=notification

**Comment by kartik1000** [comment]
@sudo-bot /deploy

File: index.php [hunk 1]
```diff
@@ -259,12 +259,8 @@
            . Url::getHiddenInputs(null, null, 4, 'collation_connection')
            . '            <label for="select_collation_connection">' . "\n"
            . '                ' . Util::getImage('s_asci')
-            . "&nbsp;" . __('Server connection collation') . "\n"
-           // put the doc link in the form so that it appears on the same line
-           . Util::showMySQLDocu('Charset-connection')
-           . ': ' .  "\n"
-           . '            </label>' . "\n"
-
+            . "&nbsp;" . __('Server connection collation :') . "\n"
```
Comment (ibennetch) [inline index.php]: There should be no space between "collation" and the colon.

File: libraries/classes/LanguageManager.php [hunk 2]
```diff
@@ -946,11 +946,8 @@ public function getSelectorDisplay($use_fieldset = false, $show_doc = true)
         // For non-English, display "Language" with emphasis because it's
         // not a proper word in the current language; we show it to help
         // people recognize the dialog
-        $language_title = __('Language')
-            . (__('Language') != 'Language' ? ' - <em>Language</em>' : '');
-        if ($show_doc) {
-            $language_title .= Util::showDocu('faq', 'faq7-2');
-        }
+        $language_title = __('Language :')
+            . (__('Language :') != 'Language :' ? ' - <em>Language</em>' : '');
```
Comment (ibennetch) [inline libraries/classes/LanguageManager.php]: Same thing here, there shouldn't be a space before the colon.

Comment (kartik1000) [reply libraries/classes/LanguageManager.php]: I'll make the same change here as well :)

File: index.php (hunk 1 above)
Comment (kartik1000) [reply index.php]: I'll make the changes :)

**Comment by kartik1000** [comment]
@ibennetch or @williamdes made the required changes, Pls review :)

**Pull request closed without merging**